neurodamus/target_manager.py

```diff
@@ -269,9 +220,100 @@ def pathways_overlap(self, conn1, conn2, equal_only=False):
             return TargetSpec(src1) == TargetSpec(src2) and TargetSpec(dst1) == TargetSpec(dst2)
         return self.intersecting(src1, src2) and self.intersecting(dst1, dst2)
 
-    def __getattr__(self, item):
-        logging.debug("Compat interface to TargetManager::" + item)
-        return getattr(self.hoc, item)
+    def getPointList(self, target, **kw):
+        return self.get_target_points(self._targets[target], self._cell_manager, **kw)
```

Comment:
Now we call the chain
```
TargetManager.getPointList(target, cell_manager) =>
     TargetManager.get_target_points(target, cell_manager =>
          target.getPointList(cell_manager)
```
Shall we remove the middle call `TargetManager.get_target_points` ?

Reply:
makes sense, but i will then rename the first call to get_target_points() and only have:
TargetManager.get_target_points(target, cell_manager) =>
          target.getPointList(cell_manager)